js/src/dapps/registry/Lookup/actions.js

```diff
@@ -31,12 +32,28 @@ export const lookup = (name, key) => (dispatch, getState) => {
     .find((f) => f.name === 'getAddress');
 
   name = name.toLowerCase();
-  dispatch(start(name, key));
+  dispatch(lookupStart(name, key));
```

Comment:
Could do with a blank line following this statement before we enter the promise block.